### PR TITLE
extending tests to include services, clusters, and templates

### PR DESCRIPTION
**Pull request by rns4731**
- Tests for services and templates
- Bug fixes

### CODE REVIEW

**Comment by rns4731** [comment]
Update - remove classname for create a cluster btn, not sure why it can't find it. Need to keep create btn and add automator btn class names because there are multiple instances of the identifiers inside on the page.

File: coopr-e2e/test/e2e/services/service-spec.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    element(by.css('#inputServiceDescription')).sendKeys('bar');
+
+    element(by.cssContainingText('.my-thing-picker a.btn-warning:not([disabled])', 'add feature')).click();
+    element(by.css('#newThingName-feature')).sendKeys('feature1');
+    element(by.partialButtonText('Add')).click();
+
+    var provides = element.all(by.repeater('name in model'));
+    expect(provides.count()).toEqual(2);
+
+    element(by.css('.add-automator-action-btn')).click();
```
Comment (gaarf) [inline coopr-e2e/test/e2e/services/service-spec.js]: by.css('.btn', 'add automator action')

File: coopr-e2e/test/e2e/templates/template-spec.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    ).toMatch(/\/templates\/create\/tab\/2$/);
+
+    element(by.css('#inputTemplateDefaultProvider option:nth-child(2)')).click();
+    element(by.cssContainingText('fieldset.active button.btn.btn-sm.btn-default', 'Next')).click();
+
+    expect(
+      browser.getLocationAbsUrl()
+    ).toMatch(/\/templates\/create\/tab\/3$/);
+
+    element(by.cssContainingText('button.action-btn:not(.ng-hide)', 'Create')).click();
```
Comment (gaarf) [inline coopr-e2e/test/e2e/templates/template-spec.js:63]: by.cssContainingText('form button[type=submit]', 'Create')

Comment (rns4731) [inline coopr-e2e/test/e2e/templates/template-spec.js:63]: There are plenty of ways to do this but it's no guarantee that it will work remotely. I'm going to send out another PR to disable to these tests.
